tools/license_header.py

```diff
@@ -100,24 +112,33 @@
                'docs/static_site/src/assets/js/clipboard.js',
                'cmake/upstream/FindCUDAToolkit.cmake',
                'cmake/upstream/select_compute_arch.cmake',
-               'src/operator/numpy/np_einsum_op-inl.h',
 
                # Licensed under 2-Clause BSD in header
                'example/ssd/dataset/pycocotools/coco.py',
 
                # Julia package metadata, generated by Pkg3.jl
                'julia/Project.toml',
-               
+
                # Licensed under Apache 2.0 license
-               'example/image-classification/predict-cpp/image-classification-predict.cc'
+               'example/image-classification/predict-cpp/image-classification-predict.cc',
+
+               # This file
+               'tools/license_header.py',
```

Comment:
Why is this file excluded? It seems to have a license header already?

Reply:
As this file contains the pattern of other license in https://github.com/apache/incubator-mxnet/blob/0db9121620a76c8be91d66ba3e6984108a452b92/tools/license_header.py#L62
And it will be considered as a multiple license case by itself, and that's why it's excluded.